docs/reference/edot-collector/components/kubeletstatsreceiver.md

```diff
@@ -0,0 +1,122 @@
+---
+navigation_title: Kubelet stats receiver
+description: The Kubelet stats receiver is an OpenTelemetry Collector component that collects node, pod, container, and volume resource metrics from the Kubernetes Kubelet.
+applies_to:
+  stack:
+  serverless:
+    observability:
+  product:
+    edot_collector:
+products:
+  - id: cloud-serverless
+  - id: observability
+  - id: edot-collector
+---
+
+# Kubelet stats receiver
+
+The Kubelet stats receiver collects Kubernetes node, pod, container, and volume metrics directly from the Kubelet API. It is enabled by default in several {{product.observability}} Kubernetes pipelines and is a core component of the {{edot}} Collector distribution.
+
+This receiver queries the Kubelet’s `/stats/summary` endpoint and converts the retrieved usage statistics into OpenTelemetry metrics.
+
+For the contrib component documentation, see the [kubeletstatsreceiver README](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/kubeletstatsreceiver).
```

Comment:
For other K8's components, our wording is a little different. Examples:

> For full contrib details, refer to the [OpenTelemetry k8s_cluster receiver documentation](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/k8sclusterreceiver)

> For full contrib details, refer to the [OpenTelemetry k8sobjects receiver documentation](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/k8sobjectsreceiver).

Also, is it safe to assume that most OTel users will know what this means and what the differences in docs are or should be?